pages/osx/security.md

```diff
@@ -13,4 +13,12 @@
 
 - Create a keychain:
 
-`security create-keychain -p {{password}} {{name.keychain}}`
+`security create-keychain -p {{password}} {{keychain.name}}`
+
+- Set a certificate to use with a website or [s]ervice by its [c]ommon name (this will fail if several certificates with the same common name exist):
+
+`security set-identity-preference -s {{URL|hostname|service}} -c {{friendly_certificate_name_in_keychain}} {{path/to/file.keychain}}`
+
+- Add a certificate from file to a [k]eychain (if -k isn't specified, the default keychain is used):
+
+`security add-certificates -k {{keychain.name}} {{path/to/file}}`
```

Comment:
Aren't certs usually `.pam` files? I'd do it like this:

```suggestion
`security add-certificates -k {{keychain.name}} {{path/to/file.pam}}`
```

Reply:
They are usually .p**e**m, .p**a**m is for sshd

Reply:
I've added the .pem extension